### PR TITLE
DEV-2802 - Add file specific tidy functions

### PR DESCRIPTION
**Pull request by nomad-software**
Made the OHIP API proto files 100% automatically generated to more easily support upcoming tickets.

### CODE REVIEW

File: cmd/tidy/global/global.go [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+	contents = ReplaceAll(contents, "message 503", "message Http503")
+
+	contents = removeDuplicateMessages(contents)
+	contents = removeStubMessages(contents)
+	contents = snakeCasePlaceholders(contents)
+
+	return contents
+}
+
+func Replace(text, old, new string, count int) string {
```
Comment (mryan321) [inline cmd/tidy/global/global.go:39]: not sure about these - why not just use the `strings` funcs straight up? 🤔

**Review by mryan321 — APPROVED** [review]
1 minor comment 😊

File: cmd/tidy/global/global.go (hunk 1 above)
Comment (nomad-software) [reply cmd/tidy/global/global.go]: Yeah it does look a bit redundant I think it's a holdover when they did a bit more. I'll remove them.

**Review by db80 — APPROVED** [review]
I love it ❤️

**Pull request merged**